src/parser/parse_tree.h

```diff
@@ -3515,7 +3515,11 @@ struct execution_state_values
 typedef struct keyword_record KEYWORD_RECORD;
 struct keyword_record
 {
+#if defined(ENABLE_UNUSED_FUNCTION)
```

Comment:
this macro should be only at the function to be used sometime in the future. I think this macro is not needed.

Reply:
This will remain as it is for now.